toptal/picasso
# fix: minWidth in Button.Action

[FX-1139](https://toptal-core.atlassian.net/browse/FX-1139)

### Description

The https://github.com/toptal/picasso/pull/1593 introduced the `Button.Action` component that had the `minWidth: 56px` setup, which was breaking the styling when there was just an icon (`16px` wide, for example) used for the action button (for example, in Accordions). Please see the demo section to see the difference.

### How to test

- visit the https://picasso.toptal.net/FX-1139-fix-action-button-style/?path=/story/components-folder--accordion and make sure that the expand button is properly aligned for Accordions.

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| ![before](https://user-images.githubusercontent.com/1390758/95160268-ae13ec80-07d2-11eb-8b1f-e82fc791b14e.png) | ![after](https://user-images.githubusercontent.com/1390758/95160277-b4a26400-07d2-11eb-9294-40704eeef369.png) |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation


</details>
